src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.ts

```diff
@@ -143,9 +164,21 @@ export class InformatieObjectViewComponent extends ActionsViewComponent implemen
                         });
                 }, 'edit'));
             }
+
+            if (this.infoObject.locked) {
+                this.menu.push(new ButtonMenuItem('actie.unlock', () => {
```

Comment:
Je mag het document alleen unlocken als je zelf de lock hebt gelegd. Dus als lockedDoor.id gelijk is aan ingelogdeMedewerker.is Of als je daar de juiste rol voor hebt maar dat ga ik nog inbouwen.